build.gradle

```diff
@@ -148,21 +152,34 @@ task copyApps( type: Copy, dependsOn: copyResources ) {
 
 task copyDist( dependsOn: copyApps )
 
+task distTar( type: Tar, dependsOn: [copyDist, jdk] ) {
+    description = 'Build full distribution tar.gz'
+    group = 'Dist'
+    baseName = targetArtifactId()
+    compression = Compression.GZIP
+    from installDir
+    into archiveBase
+}
+
 task distZip( type: Zip, dependsOn: [copyDist, jdk] ) {
-    description = 'Build full distribution zip.'
+    description = 'Build full distribution zip'
     group = 'Dist'
     baseName = targetArtifactId()
     from installDir
     into archiveBase
 }
 
-build.dependsOn distZip
+if ( isLinux() ) {
+    build.dependsOn distTar
```

Comment:
But it would make zip distribution not available for linux.
For linux we need both zip and tar.gz.
For others - only zip